### DISCUSSION TITLE
Add event highlights to date picker

### DISCUSSION THREAD

**Discussion by patrickwasp** [zauberzeug/nicegui#792] [Ideas / Feature Requests]
Not sure if this is already an option, but I couldn't find it in the documentation. 

Quasar has the option of adding events to the date picker. Can this be added to NiceGUI?

https://quasar.dev/vue-components/date#highlighting-events

**Comment by falkoschindler** [comment]
You can set events using Quasar props:
```py
ui.date(value='2023-04-01').props('''events="['2023/04/05', '2023/04/06', '2023/04/09', '2023/04/23']"''')
```

Or a bit simpler using an f-string and less complicated quotes:
```py
events = ['2023/04/05', '2023/04/06', '2023/04/09', '2023/04/23']
ui.date(value='2023-04-01').props(f'events="{events}"')
```
